sklearn/linear_model/_ridge.py

```diff
@@ -2050,23 +2064,35 @@ def fit(self, X, y, sample_weight=None):
                 if self.store_cv_values:
                     self.cv_values_[:, i] = predictions.ravel()
 
+                score_params = params or {}
```

Comment:
I originally added the get_metadata_routing and process_routing in this private class. But I think we need to actually implement routing inside RidgeCV and RidgeClassifierCV otherwise the default requests test fails for both. Are you instead suggesting to also implement routing here?

Reply:
Oh I see. And other classes don't inherit from this class. So I'd call the argument here `score_params` and make sure it's always passed.

Reply:
I adjusted the name of the parameter. I was not clear about how to ensure that it is always passed?